header.h

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+#include <string>
+
+using namespace std;
+
+class BankAccount
+{
+private:
+
+
+    string Owner_name;
+    string Replenish_balance;
+    string Withdraw_cash;
+
+public:
+
+    int Balance;
+    int ID;
+    double amoumt;
+    int amount;
+    int Getid()
+    {
+        return ID;
+    }
+
+    string Getowner_name()
+    {
+        return Owner_name;
+    }
+
+    int Getbalance()
+    {
+        return Balance;
+    }
+
+    string GetReplenish_balance()
+    {
+        return Replenish_balance;
+    }
+
+    string GetWithdraw_cash()
+    {
+        return Withdraw_cash;
+    }
+
+
+    BankAccount()
+    {
+    }
+
+    BankAccount(int id, string ownername, int balance, string  withdraw_cash)
+    {
+        ID = id;
+        Owner_name = ownername;
+        Balance = balance;
+        Withdraw_cash = withdraw_cash;
+    }
```

Comment:
It is not necessary but use this
BankAccount(int id, string owner_name, int balance, string  withdraw_cash)
    {
        this->id = id;
        this->owner_name = owner_name;
        this->balance = balance;
        this->withdraw_cash = withdraw_cash;
    }